src/mlstacks/terraform/aws-remote-state/main.tf

```diff
@@ -0,0 +1,49 @@
+provider "aws" {
```

Comment:
Might be worth extending with the Bucket Owner full control enforcement option - comes very handy in cross-account settings.
https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/s3_bucket_ownership_controls